### ISSUE TITLE
Upgrade timezone to support newer minor versions

### ISSUE THREAD

**Issue by mauriziopinotti** [syncfusion/flutter-widgets#2015]
### Bug description

In `packages/syncfusion_flutter_calendar/pubspec.yaml`, `timezone` is depended upon as 

```yaml
timezone: 0.9.3
```

However, `timezone` updates are rather frequent and only contains small fixes to the zone list.
Unfortunately, depending on a fixed version prevents other apps to stay up to date:
```
Because syncfusion_flutter_calendar >=25.2.4 depends on timezone 0.9.3 and easyhour_app depends on timezone ^0.9.4, syncfusion_flutter_calendar >=25.2.4 is forbidden.
```
Recently it was changed from 0.9.2 to 0.9.3 (see https://github.com/syncfusion/flutter-widgets/issues/1834) but without the caret symbol.

I would propose to change the dependency to `^0.9.4` so that we're good for a while.

### Steps to reproduce

```yaml
dependencies:
  syncfusion_flutter_calendar: ^26.2.8
  timezone: ^0.9.4
```

### Code sample

```yaml
dependencies:
  syncfusion_flutter_calendar: ^26.2.8
  timezone: ^0.9.4
```

### Screenshots or Video

N/A

### Stack Traces

```
Because syncfusion_flutter_calendar >=25.2.4 depends on timezone 0.9.3 and easyhour_app depends on timezone ^0.9.4, syncfusion_flutter_calendar >=25.2.4 is forbidden.
```

### On which target platforms have you observed this bug?

Android, iOS, Web

### Flutter Doctor output

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.24.0, on Ubuntu 22.04.4 LTS 5.15.0-117-generic, locale en_US.UTF-8)
[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0-rc2)
[✓] Chrome - develop for the web
[✓] Linux toolchain - develop for Linux desktop
[✓] Android Studio (version 2024.1)
[✓] IntelliJ IDEA Ultimate Edition (version 2024.2)
[✓] VS Code (version 1.92.0)
[✓] Proxy Configuration
[✓] Connected device (3 available)
[✓] Network resources

• No issues found!
```

**Comment by mauriziopinotti** [comment]
> Hi @mauriziopinotti,
> 
> We have updated the [timezone](https://pub.dev/packages/timezone/versions/0.9.4) package from version [0.9.3](https://pub.dev/packages/timezone/versions/0.9.3) to [0.9.4](https://pub.dev/packages/timezone/versions/0.9.4) for the [syncfusion_flutter_calendar](https://pub.dev/packages/syncfusion_flutter_calendar/versions/26.2.8). We will include this version update in our upcoming weekly patch release, which is expected to be rolled out by August 27, 2024. We will notify you once the release is available. We appreciate your patience in the meantime.
> 
> Regards,
> 
> Preethika Selvam.

Looks like the released has been published (26.2.11) but timezone is still set to 0.9.3 :cry:

**Comment by mauriziopinotti** [comment]
Thanks, that worked now! :+1:
